### PR TITLE
The View: JavaScript Capstone Project

### PR DESCRIPTION
**Pull request by sireclifford**
- Use ES6 syntax.
- Use ES6 modules.
- Use callbacks and promises.
- Use WebPack.
- Apply JavaScript best practices and language style guides in code.
- Use AAA pattern for unit tests.
- Write units tests for a JavaScript app.
- Follow Gitflow.
- Solve simple git conflicts.
- Send and receive data from an API.
- Use API documentation.
- Understand and use JSON.
- Make JavaScript code asynchronous.
- Perform a code review for a team member.

## Home Page
Screenshot 2022-07-29 at 1 07 58 PM

## Movie Detail
Screenshot 2022-07-29 at 1 08 13 PM

## Movie Detail with comment
Screenshot 2022-07-29 at 1 08 27 PM

## Presentation Link
https://www.loom.com/share/7cd4da981a084827aeba2e7274a5f121

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <title>The View | JS Capstone</title>
+<script defer src="runtime.bundle.js"></script><script defer src="main.bundle.js"></script></head>
+
+<body>
+  <div id="app">
+    <header>
+      <img class="logo" src="assets/images/the-view-logo.png" alt="logo">
+      <h1 class="page-title">Popular Movies (6)</h1>
```
Comment (vechicin) [inline dist/index.html:16]: You should implement a counter for the number of items on the home screen.

Screen Shot 2022-07-29 at 8 28 59

After you implement it, don't forget you need to test it as well.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @sireclifford and @anasbassi ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job implementing most of the required features for this project!

## Required Changes ♻️

- [ ] I can't find the link to the video presentation on your PR description nor on your README. Please make sure you provide the link to the video presentation.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (sireclifford) [reply dist/index.html:16]: Counter is implemented in the js file. It is dynamic and not static

**Review by mukhammadrizooff — APPROVED** [review]
Hi Team, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**